Pages/Index.cshtml

```diff
@@ -3,21 +3,26 @@
 @model IndexModel
 @{
     ViewData["Title"] = "Sips and Bites Navigator";
-    var brand = ViewData["Brand"];
-    List<Brewery> breweries = (List<Brewery>) ViewData["Breweries"];
+    var siteBrand = ViewData["Brand"];
+    List<Brewery> breweryList = (List<Brewery>) ViewData["Breweries"];
 }
 
 <div class="text-center">
-    <div class="header">
-        <h1 class="display-4">Welcome to @brand</h1>
+    <header class="header">
+        <h1 class="display-4">Welcome to @siteBrand</h1>
```

Comment:
Avoids ambiguity and aligns with the naming style of collections.

Reply:
If using header in one instance, you can apply CSS rules to this HTML element, and disregard the class.